bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/EntryLocationIndex.java

```diff
@@ -204,46 +202,21 @@ public void removeOffsetFromDeletedLedgers() throws IOException {
                 firstKeyWrapper.set(ledgerId, 0);
                 lastKeyWrapper.set(ledgerId, Long.MAX_VALUE);
 
-                Entry<byte[], byte[]> firstKeyRes = locationsDb.getCeil(firstKeyWrapper.array);
-                if (firstKeyRes == null || ArrayUtil.getLong(firstKeyRes.getKey(), 0) != ledgerId) {
-                    // No entries found for ledger
-                    log.info("No entries found for ledger {}", ledgerId);
-                    continue;
-                }
-
-                long firstEntryId = ArrayUtil.getLong(firstKeyRes.getKey(), 8);
-                long lastEntryId = getLastEntryInLedgerInternal(ledgerId);
-                log.info("Deleting index for ledger {} entries ({} -> {})",
-                        new Object[] { ledgerId, firstEntryId, lastEntryId });
+                batch.deleteRange(firstKeyWrapper.array, lastKeyWrapper.array);
```

Comment:
nit: Could we please add in at least a debug log similar to what we had above.  We don't have to log every entry that we delete, but at least log the range we are deleting.